crates/ruma-federation-api/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ Breaking changes:
     `SpaceHierarchyParentSummaryInit` was removed.
   - `SpaceHierarchyChildSummary` was replaced by `RoomSummary` and
     `SpaceHierarchyChildSummaryInit` was removed.
+- Merge the `knock` module into `membership`, and rename `create_knock_event_template` and
+  `send_knock` to `prepare_knock_event` and `create_knock_event` respectively for consistency.
+- Use `StrippedState` instead of `Raw<AnyStrippedStateEvent>`, to allow non-stripped events to be
+  represented for `create_invite` and `send_knock`.
```

Comment:
Since `send_knock` was renamed, I believe that the changelog should use the new name here.